10-angular/introduction_to_angular/readme.md

```diff
@@ -195,17 +189,17 @@ In `index.html`:
 </div>
 ```
 
-> we used the `ng-controller` directive here in order to instantiate our controller in the DOM. In the same way that `ng-app` established the domain of the js functionality in the html element, the `ng-controller` establishes the domain in this div.
+> we used the [`ng-controller` directive](https://docs.angularjs.org/api/ng/directive/ngController) here in order to instantiate our controller in the DOM. In the same way that `ng-app` established the domain of the js functionality in the html element, the `ng-controller` establishes the domain in this div.
 
 > `todosCtrl` is an instance of our controller. `ng-controller` gives this whole div access to all the values and methods defined in that controller. This instance of a controller is called a "ViewModel".
```

Comment:
Yeah, I tend to agree, `ViewModel` seems better then `Ctrl`. I've seen alot of tutorials, at the very least i'd identify this to the class that they may see it in stack overflows/google/ etc.